sql/ddl/data-types/encoding.md

```diff
@@ -40,14 +40,14 @@ When it comes to storing and manipulating string fields, one question that comes
 
 You can see the **encoding** of a system as a function that maps groups of bits to a character. One of the first encodings was ASCII: it maps characters to a numeric value and it can represent up to 127 characters - 'A' maps to 65, 'a' to 97, etc. Extended ASCII can be used to represent up to 255 characters.
 
-As you can imagine, 255 characters isn't nowhere near enough to map every symbol of every alphabet in the world. For reference, only in the first emoji tab ('Smileys and People') there are around 180 emoticons. This was an issue that had to be dealt with quickly.
+As you can imagine, 255 characters is nowhere near enough to map every symbol of every alphabet in the world. For reference, there are around 180 emoticons in only the first emoji tab ('Smileys and People'). This was an issue that had to be dealt with quickly.
```

Comment:
what do you think about this phrasing? (wanted to remove 'only' as it sounded a bit weird)

```suggestion
As you can imagine, 255 characters are nowhere near enough to map every symbol of every alphabet in the world. For reference, there are around 180 emoticons in the 'Smileys and People' emoji tab. This was an issue that had to be dealt with quickly.
```

Reply:
looks good